routers/org/org.go

```diff
@@ -1,4 +1,5 @@
 // Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2018 The Gitea Authors. All rights reserved.
```

Comment:
@techknowlogick 2019

Reply:
This file was modified in 2018, but the copyright wasn't added.